public/style.css

```diff
@@ -3195,7 +3195,7 @@ grammarly-extension {
 
 #bulk_tag_popup,
 #dialogue_popup {
-    width: 500px;
+    width: 750px;
```

Comment:
Even if this *only* applies to the old legacy popups, we shouldn't just change the base width like this. This applies to all popups using that.
We have the `wider_dialogue_popup` CSS class applied for popups that should be wider. That one is used for the "Import Tags" popup too, for example.

Some of these days we should for sure update the bulk edit popups to the new popup functionality. Until then, could you check if we can find a way to just add the CSS class to the shadow popup for this?

I don't want to change the base width.